packages/portal/src/server-middleware/api/index.js

```diff
@@ -35,8 +35,11 @@ app.get('/debug/memory-usage', debugMemoryUsage);
 import cache from './cache/index.js';
 app.get('/cache/*', (req, res) => cache(req.params[0], runtimeConfig.redis)(req, res));
 
-import jiraServiceDesk from './jira/service-desk.js';
-app.post('/jira/service-desk', (req, res) => jiraServiceDesk(runtimeConfig.jira)(req, res));
+import jiraServiceDesk from './jira-service-desk/feedback.js';
+app.post('/jira-service-desk/feedback', (req, res) => jiraServiceDesk(runtimeConfig.jira)(req, res));
```

Comment:
Changing the URL to this is fine, but we need to also keep supporting the old URL for a while, in case users have the website open already when we deploy a new version and try to submit feedback. Keep the new one, but restore the old one too, and give it a TODO note for future removal.

Reply:
Done.